python_modules/dagster/dagster/_core/definitions/freshness_checks/last_update.py

```diff
@@ -0,0 +1,63 @@
+import datetime
+from typing import Optional, Sequence, Union
+
+from dagster._annotations import experimental
+from dagster._core.definitions.asset_check_spec import AssetCheckSeverity
+
+from ..asset_checks import AssetChecksDefinition
+from ..assets import AssetsDefinition, SourceAsset
+from ..events import CoercibleToAssetKey
+from .utils import (
+    DEFAULT_FRESHNESS_CRON_TIMEZONE,
+    DEFAULT_FRESHNESS_SEVERITY,
+    build_freshness_checks_for_assets,
+)
+
+
+@experimental
+def build_last_update_freshness_checks(
+    *,
+    assets: Sequence[Union[CoercibleToAssetKey, AssetsDefinition, SourceAsset]],
+    lower_bound_delta: datetime.timedelta,
```

Comment:
What timezone is the delta math done in? `timedelta(days=1)` can be 23 hours in certain time zones at certain times.

Reply:
Changed things here to make sure we keep current time a pure unix timestamp, and perform timezone conversion when necessary.

Reply:
Just to follow up as well, I added a test for the complex timezone interaction case.

Reply:
So the answer to "What timezone is the delta math done in?" is "always UTC"?